common-rest/src/main/java/org/uniprot/api/rest/service/BasicSearchService.java

```diff
@@ -102,7 +103,17 @@ public R getEntity(String idField, String value) {
     public QueryResult<R> search(SearchRequest request) {
         SolrRequest solrRequest = createSearchSolrRequest(request);
         QueryResult<D> results = repository.searchPage(solrRequest, request.getCursor());
-        Stream<R> converted = results.getContent().map(entryConverter).filter(Objects::nonNull);
+        Stream<R> converted;
+        if (LIST_MEDIA_TYPE_VALUE.equals(request.getFormat())) {
+            converted =
+                    results.getContent()
+                            .map(Document::getDocumentId)
+                            .map(this::mapToThinEntry)
+                            .filter(Objects::nonNull);
+        } else {
+            converted = results.getContent().map(entryConverter).filter(Objects::nonNull);
```

Comment:
This else block could call super method (extracted from line 105) that is defined in the BasicSearchService